cfme/v2v/migrations.py

```diff
@@ -0,0 +1,329 @@
+
+
+class InfraMappingWizard(View):
+    """Infrastructure Mapping Wizard Modal Widget.
+    Usage:
+        fill: takes values of following format:
+            {
+                'general_view':
+                    {
+                        'name':'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                        'description':fauxfactory.gen_string("alphanumeric",length=50)
+                    },
+                'clusters_view':
+                    {
+                        'mappings': [
+                            {
+                                'sources':['Datacenter \ Cluster'],
+                                'target':['Default \ Default']
+                            }
+                        ]
+                    },
+                'datastores_view':{
+                    'Cluster (Default)': {
+                       'mappings':[
+                            {
+                                'sources':['NFS_Datastore_1','iSCSI_Datastore_1'],
+                                'target':['hosted_storage']
+                            },
+                            {
+                                'sources':['h02-Local_Datastore-8GB', 'h01-Local_Datastore-8GB'],
+                                'target':['env-rhv41-01-nfs-iso']
+                            }
+                        ]
+                   }
+                },
+                'networks_view':{
+                    'Cluster (Default)': {
+                        'mappings': [
+                            {
+                                'sources':['VM Network','VMkernel'],
+                                'target':['ovirtmgmt']
+                            },
+                            {
+                                'sources':['DPortGroup'],
+                                'target':['Storage VLAN 33']
+                            }
+                        ]
+                    }
+                }
+            }
+    """
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    general_view = View.nested(InfraMappingWizardGeneralView)
+    clusters_view = View.nested(InfraMappingWizardClustersView)
+    datastores_view = View.nested(InfraMappingWizardDatastoresView)
+    networks_view = View.nested(InfraMappingWizardNetworksView)
+    results_view = View.nested(InfraMappingWizardResultsView)
+
+    def after_fill(self, was_change):
+        if was_change:
+            self.results_view.next_btn.click()
+
+
+class MigrationDashboardView(BaseLoggedInPage):
+    create_infrastructure_mapping = Text(locator='(//a|//button)'
+        '[text()="Create Infrastructure Mapping"]')
+    create_migration_plan = Text(locator='(//a|//button)[text()="Create Migration Plan"]')
+
+    @property
+    def is_displayed(self):
+        return self.navigation.currently_selected == ['Compute', 'Migration']
+
+
+class AddInfrastructureMappingView(View):
+    wizard_form = InfraMappingWizard()
```

Comment:
Consider just using `form` here, for some consistency in our Views with forms and their widget access. That way our implementations are more consistently `view.form.fill()`